cmd/operator-sdk/bundle/cmd.go

```diff
@@ -27,6 +27,7 @@ type bundleCmd struct {
 	defaultChannel string
 	channels       string
 	generateOnly   bool
+	json           bool
```

Comment:
Something like the scorecard's [`outputFormat`](https://github.com/operator-framework/operator-sdk/blob/2d1474e/cmd/operator-sdk/alpha/scorecard/cmd.go#L125) would be better, and default to `text`.

Reply:
We have just 2 types JSON and not JSON and we have no plain do add more types. So, why increase the complexity and allow any text for after check if the text informed is or not valid and etc. IHMO it is much easier to be used as if I want a JSON result than I just use the flag `--json` as it usually is other tools as well.

Reply:
What if we want to add YAML support in the future? Also we should align UX across all commands.

Reply:
regards `What if we want to add YAML support in the future?` I think it would be an premature optimization.   

regards `Also we should align UX across all commands.` I agree. But, we have just only 1 place with which also could be simplified as done here. 

So, if you do not mind, if more at least one person also agrees that it should be done be with outputFormat as suggested I am happy in change. Otherwise, I'd prefer to keep it as simple as possible.

Reply:
I agree with @estroz. We know this is the first non-text output format we want to support and we know there will be a follow-on. Therefore there will be _at least_ 3 output formats at some point while we transition from the alpha json output to something else.

I would vote for `-o,--output` being the flag name (mirroring `kubectl`) and for this particular format to be called `json-alpha1` (or similar).

That way, once the bundle validation API is improved, we can deprecate `json-alpha1` and add a new value that users can transition to so we don't break them abruptly.

Reply:
I think I agree with @estroz here and I don't think it's premature optimization. I think it's more not painting ourselves in a corner. If we have `--json` now. Then want yaml, would that be `--yaml`? What about plain text? '--plain`? Refactoring CLI flags is more cumbersome.

Reply:
All done 👍 tks for the feedback.